tutorials/lunar_lander.ipynb

```diff
@@ -555,17 +555,25 @@
     }
    ],
    "source": [
+    "from dask.distributed import Client, LocalCluster\n",
```

Comment:
In the list of pip installs at the top, add `dask` and `distributed`.

Also, in the list of imports at the top, put `import dask.distributed` and remove `import multiprocessing` -- note that the import should go in the second paragraph of imports since it's an external import, and also put it in alphabetical order.